OneSignalSDK/onesignal/src/main/java/com/onesignal/OSUtils.java

```diff
@@ -439,4 +443,16 @@ static boolean hasConfigChangeFlag(Activity activity, int configChangeFlag) {
       }
       return hasFlag;
    }
+
+   static @NonNull Collection<String> extractStringsFromCollection(@Nullable Collection<Object> collection) {
+      Collection<String> result = new ArrayList<>();
+      if (collection == null)
+         return result;
+
+      for (Object value : collection) {
+         if (value instanceof String)
```

Comment:
maybe we could add an else and log that "This is an invalid type" and it only excepts Strings

Reply:
Done, added a warning to `removeTriggersForKeysFromJsonArrayString`.
Kept `extractStringsFromCollection` the same as wanted to keep this a pure utility method.